week5.3-ex1.py

```diff
@@ -0,0 +1,56 @@
+START_PATH = r"C:\Users\97252\python\Notebooks\week05\\"
+PATH = r"resources\logo.jpg"
+
+
+def open_and_read_from_file():
+    """
+    This generator open binary file according the path up
+    :return: 1024 bytes from the file every yield
+    """
+    try:
+        with open(START_PATH+PATH, 'rb') as input_file:
+            for binary_text in iter(lambda: input_file.read(1024), b''):
+                yield binary_text
+    except IOError:
+        print("Error: can't open the file")
+
+
+def is_secret_message(text: bytes) -> (bool, str):
+    """
+    This function get text and checks if it is end with just lower characters, at least 5
+    :param text: the text is bytes, so we will do casting to str
+    :return: bool- if is just lower characters and at least 5. And if so, return this text- str
```

Comment:
Why do you need to return this bool? cant you just return an empty string\None or the text?